RiotSwiftUI/Modules/Authentication/ServerSelection/MockAuthenticationServerSelectionScreenState.swift

```diff
@@ -38,7 +38,7 @@ enum MockAuthenticationServerSelectionScreenState: MockScreenState, CaseIterable
         let viewModel: AuthenticationServerSelectionViewModel
         switch self {
         case .matrix:
-            viewModel = AuthenticationServerSelectionViewModel(homeserverAddress: "https://matrix.org",
+            viewModel = AuthenticationServerSelectionViewModel(homeserverAddress: "matrix.org",
```

Comment:
This changed in the Coordinator as is now gives the view model an address that has already been formatted.